addons/crm/data/crm_lead_demo.xml

```diff
@@ -871,6 +844,21 @@ Andrew</p>]]></field>
             <field name="source_id" ref="utm.utm_source_search_engine"/>
         </record>
 
+        <record id="msg_case31_1" model="mail.message">
+            <field name="model">crm.lead</field>
```

Comment:
Consider ordering keys alphabetically (I know this file is not ordered but hey, let us introduce order in the universe one demo data at a time !)